Dockerfile

```diff
@@ -1,18 +1,22 @@
 ARG BCI_IMAGE=registry.suse.com/bci/bci-busybox
 ARG GO_IMAGE=rancher/hardened-build-base:v1.21.8b1
 ARG ARCH="amd64"
+
+# Image that provides cross compilation tooling.
+FROM --platform=$BUILDPLATFORM rancher/mirrored-tonistiigi-xx:1.3.0 as xx
```

Comment:
Is it possible to keep track of that `1.3.0` version with updateCLI?

Reply:
It should be possible. Maybe in another PR?

Reply:
yes!